### PR TITLE
Convert template engine to Text::Xslate

### PR DESCRIPTION
**Pull request by haarg**
WIP of converting the templates to use Text::Xslate

### CODE REVIEW

File: lib/MetaCPAN/Web/Controller/Root.pm [hunk 1]
```diff
@@ -62,7 +67,9 @@ sub not_found : Private {
     $c->cdn_never_cache(1);
 
     $c->stash( {
-        template     => 'not_found.html',
+        #template     => 'not_found.html',
```
Comment (oalders) [inline lib/MetaCPAN/Web/Controller/Root.pm:70]: Left over debugging?

File: lib/MetaCPAN/Web/Controller/Root.pm [hunk 2]
```diff
@@ -72,15 +79,35 @@ sub internal_error : Private {
     my ( $self, $c ) = @_;
     $c->cdn_never_cache(1);
 
-    $c->stash( { template => 'internal_error.html' } );
+    $c->stash( {
+            #template => 'internal_error.html',
```
Comment (oalders) [inline lib/MetaCPAN/Web/Controller/Root.pm:83]: Debugging?

**Review by oalders — APPROVED** [review]
From what I can tell, it looks great.

**Comment by oalders** [comment]
I think this will have to go out of draft mode in order for CI to run.

**Comment by haarg** [comment]
All of the templates are now converted. Still needs substantial testing and review.

**Comment by haarg** [comment]
Tests are passing.

**Comment by oalders** [comment]
I'm going to temporarily move this out of draft mode to see if we can get the Docker build to run.

**Comment by oalders** [comment]
It doesn't show up as an available branch for the workflow. It may need to be rebased. Probably a good idea since there's the one merge conflict.

**Comment by oalders** [comment]
Oh, it's because you've forked.

**Comment by haarg** [comment]
Replaced by #2490

**Pull request closed without merging**